src/analysis/inner_product_space/basic.lean

```diff
@@ -1170,6 +1170,17 @@ begin
     simp only [linear_map.zero_apply, inner_zero_left] }
 end
 
+/--
+If `⟪S x, x⟫_ℂ = ⟪T x, x⟫_ℂ` for all x, then S = T.
```

Comment:
Can you please modify the docstring so that it matches the statement? I would also prefer something more in the direction of "Two linear maps are equal if and only if.."

Reply:
Thanks for the feedback @mcdoll. I tried to mimic the Lemma `inner_map_self_eq_zero` in this PR. Should that one also be updated for consistency?

Reply:
Yeah, that docstring does not seem to be helpful either. It would be great if you change that too.

Reply:
Ok, updated, would this be better @mcdoll ?